app/src/main/java/org/mozilla/reference/browser/BrowserApplication.kt

```diff
@@ -23,6 +33,14 @@ class BrowserApplication : Application() {
         }
     }
 
+    private fun isChildProcess(): Boolean {
+        val pid = android.os.Process.myPid()
+        val activityManager = this.getSystemService(Context.ACTIVITY_SERVICE) as ActivityManager
+        val processes = activityManager.runningAppProcesses
+        val processName = processes.firstOrNull { it.pid ==pid }?.processName
```

Comment:
nit: why not just

```kt
val activityManager = this.getSystemService(Context.ACTIVITY_SERVICE) as ActivityManager
val info = activityManager.runningAppProcesses.firstOrNull {
    it.pid == Process.myPid()
}
return info?.processName?.contains(":") ?: false
```

Reply:
No reason. Just found this easier to read.

Reply:
OK, updated as well.